epics/ca.py

```diff
@@ -26,10 +26,10 @@
 
 from math import log10
 from pkg_resources import resource_filename
+from copy import deepcopy
 
-from .utils import (STR2BYTES, BYTES2STR, NULLCHAR_2,
-                    strjoin, memcopy, is_string, is_string_or_bytes,
-                    ascii_string, clib_search_path)
+from .utils import (str2bytes, bytes2str, strjoin,
```

Comment:
Name changes are a step in the right direction, but you may consider back-compat for the old names. Even if you don't consider these part of the public API, I'd bet that some out there are using it:

```
# Back-compat; deprecated and to be dropped in pyepics x.y.z
STR2BYTES = str2bytes
```

Reply:
@klauer   Yeah, I think you're probably right, I will add back-compatible aliases for all of those legacy names in `utils.py` where they can enjoy a long and boring life now that we know we don't really need them.